source/sass/buyers-guide/views/homepage.scss

```diff
@@ -49,6 +49,24 @@
         padding: $padding-y $padding-x;
         width: calc((100% - #{$row-total-gutter-width}) / #{$items-per-row});
 
+        &.draft {
+          border: 1px solid #ff000054;
```

Comment:
this HEX code looks too long for me?

Reply:
last two digits are the transparency.

Reply:
ah that's something new to me! 
ignore all my other comment then :D